internal/manifest/oci_index.go

```diff
@@ -79,6 +82,18 @@ func (index *OCI1IndexPublic) UpdateInstances(updates []ListUpdate) error {
 	return index.editInstances(editInstances)
 }
 
+func annotationsToCompressionAlgorithmNames(annotations map[string]string) []string {
+	result := []string{}
```

Comment:
Nit: pre-allocate the slice with size 1.

Reply:
Done.